.github/actions/test-asdf-caf-scripts/action.yaml

```diff
@@ -0,0 +1,50 @@
+name: 'Test ASDF CAF Scripts'
+description: 'Install ASDF and required plugins and test functionality of CAF scripts'
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Install ASDF
+      if: ${{ inputs.install_asdf == 'true' }}
+      shell: bash
+      run: |
+        git clone https://github.com/asdf-vm/asdf.git ${{ inputs.asdf_dir }} --branch ${{ inputs.asdf_version }}
+
+    - name: Install Plugins
+      shell: bash
+      run: |
+        . "${{ inputs.ASDF_DIR }}/asdf.sh"
+        asdf plugin add caf-scripts "https://github.com/weareadaptive/asdf-caf-scripts"
+        echo -e "\ncaf-scripts ref:${{ github.sha }}" >> .tool-versions
+        cut -d' ' -f1 .tool-versions | xargs -I{} asdf plugin-add {} || true
+        asdf install
+
+    - name: Test caf_create_local_k8s
+      shell: bash
+      run: |
+        . "${{ inputs.ASDF_DIR }}/asdf.sh"
+        caf_create_local_k8s.sh
+
+    - name: Test caf_route_to_local_k8s
+      shell: bash
+      run: |
+        # Install ingress-nginx
+        helm upgrade --install ingress-nginx ingress-nginx --repo https://kubernetes.github.io/ingress-nginx --namespace nginx-ingress --create-namespace
+
+        . "${{ inputs.ASDF_DIR }}/asdf.sh"
+        caf_route_to_local_k8s.sh &
+        curl -s --connect-timeout 5 --retry 5 --retry-delay 5 kube-dns.kube-system.svc.cluster.local:9153/metrics
```

Comment:
This is nice way to test but I think the metrics output itself doesn't give us much and is a bit cluttered. 
Can we perhaps do a silent curl and echo something like "/metrics endpoint is reachable at kube-dns.etc.etc. Successfully tested caf_route_to_local_k8s.sh".

Reply:
Thanks for the recommendation. I updated my test with this in mind